charts/verdaccio/templates/NOTES.txt

```diff
@@ -3,9 +3,9 @@
 {{- if .Values.ingress.enabled }}
   {{- range $host := .Values.ingress.hosts }}
   {{- if $tls }}
-  https://{{ $host }}
+  https://{{ include "tplvalues.render" (dict "value" $host "context" $) }}
```

Comment:
You use tplvalues.render a lot, and I can't find docs on what this would do. I've seen something related in bitnami's charts, but they have a "common" chart that it's including from, and I have not seen verdaccio have a tplvalues chart.

Can you educate me on where this is going to pull this function from?

Reply:
Sure thing, `tplvalues.render` was added to this chart in commit [`71b994e`](https://github.com/verdaccio/charts/commit/71b994efe5f9c700669aefa25792a585263cb849)

If a `value` is a String it passes it through Helm's templating engine to render it (for example: `verdaccio-{{ .Release.Namespace }}.example.com` would render as `verdaccio-default.example.com`

Otherwise, if the `value` is not a string, the `toYaml` function effectively converts it to a string, for example:
```yaml
extraInitContainers:
  - name: example
    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```
Is just as valid and will return the same output as:
```yaml
extraInitContainers: |-
  - name: example
    image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

The reason I use it extensively is because I figured it was the standard in this chart for templating values and I wanted to stick to the standard.
I guess for use cases like this (ingress hosts) where the `value` will **always** be a string, it makes more sense to just use `tpl $host $` instead